src/test/java/com/amihaiemil/docker/TcpDockerTestCase.java

```diff
@@ -171,4 +171,18 @@ public void returnsVolumes() {
             Matchers.notNullValue()
         );
     }
+
+    /**
+     * TcpDocker can return Networks.
+     */
+    @Test
+    public void returnsNetworks() {
+        MatcherAssert.assertThat(
+                new TcpDocker(
+                        Mockito.mock(HttpClient.class),
+                        URI.create("http://localhost")
+                ).networks(),
+                Matchers.notNullValue()
```

Comment:
@lumimul Can you please indent the lines 4 spaces to the left?

Reply:
done

Reply:
done